smartgrid-prediction/frontend/src/main/java/io/pivotal/demo/smartgrid/frontend/TimeSeriesDataRequest.java

```diff
@@ -0,0 +1,63 @@
+package io.pivotal.demo.smartgrid.frontend;
+
+import io.pivotal.demo.smartgrid.frontend.timeseries.DataPointResolution;
+
+public class TimeSeriesDataRequest {
```

Comment:
This is really about showcasing the smartgrid demo. There are a *lot* of minor details we could "optimize", but IMHO the less "magic" there is the better. The focus of the demo is Spring XD and the ingestion streams